internal/store/files/WorldGeosForPlaceRecognitionAlternateNames.csv

```diff
@@ -272,16 +265,11 @@ geoId/05117,Prairie
 geoId/05119,Pulaski
 geoId/05121,Randolph
 geoId/05123,St. Francis
-geoId/05125,Saline
 geoId/05127,Scott
 geoId/05129,Searcy
 geoId/05133,Sevier
-geoId/05135,Sharp
-geoId/05137,Stone
-geoId/05139,Union
 geoId/05141,Van Buren
 geoId/05143,Washington
```

Comment:
remove Washington?

Reply:
I tried the query that is fine.  Matches state